ui/pages/Project/selectors.js

```diff
@@ -373,7 +373,12 @@ const ANALYSED_BY_CATEGORY_OPTION_LOOKUP = CATEGORY_FAMILY_FILTERS[FAMILY_FIELD_
 )
 
 const analysedByFilters = (filter, analysedByOptions) => {
-  const filters = filter.map(val => FAMILY_FILTER_LOOKUP[val]).filter(val => val)
+  const filters = []
+
+  const otherFilters = filter.map(val => FAMILY_FILTER_LOOKUP[val]).filter(val => val)
+  if (otherFilters.length) {
+    filters.push(otherFilters)
```

Comment:
It looks like the `filters` becomes a kind of filter group. You may want to rename it.